superset/assets/javascripts/SqlLab/components/SouthPane.jsx

```diff
@@ -24,6 +24,36 @@ const defaultProps = {
 };
 
 class SouthPane extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      innerTabHeight: this.getInnerTabHeight(),
+    };
+  }
+  getInnerTabHeight() {
+    // hack to get height the tab container so it can be fixed and scroll in place
+    // calculate inner tab height
+
+    // document.getElementById('brace-editor').getBoundingClientRect().height;
+    const sqlEditorHeight = 192;
```

Comment:
Hard coding all of these values seems like it sin't ideal. Airpal had a feature to change the sizes of its SqlEditor for example and I could see us add something like that in the future.

Wouldn't using something like `$('#SqlEditor').height()` work?